src/sentry/static/sentry/app/components/dropdownButton.tsx

```diff
@@ -22,31 +22,43 @@ type Props = React.ComponentProps<typeof Button> & {
    * Should the bottom border become transparent when open?
    */
   hideBottomBorder?: boolean;
+  /**
+   * Button color
+   */
+  priority?: 'primary' | 'form';
+  /**
+   * Forward a ref to the button's root
+   */
   forwardedRef?: React.Ref<typeof Button>;
 };
 
 const DropdownButton = ({
-  isOpen,
   children,
   forwardedRef,
   prefix,
+  isOpen = false,
   showChevron = false,
-  hideBottomBorder = true,
```

Comment:
We shouldn't change this default

Reply:
@priscilawebdev I'm fixing another Dropdown issue that requires your changes, I think we'll want to cherry pick this commit into your PR. I've made some comments in the commit too: https://github.com/getsentry/sentry/commit/6b24ffbe52b726c9835e4ffbf707d1ee3b37fbf6

Reply:
oh good catch...let me update this